lib/std/Target.zig

```diff
@@ -8,6 +8,7 @@ os: Os,
 abi: Abi,
 ofmt: ObjectFormat,
 dynamic_linker: DynamicLinker = DynamicLinker.none,
+page_size: ?usize = null,
```

Comment:
I don't think page_size should be part of Target.

Reply:
What would the alternative be? Move it into OS so then CPU model and OS can determine the actual page size?

Reply:
It should be determined at runtime

Reply:
Then how should we get around the slices which rely on the page size being comptime?

Reply:
Make them stop doing that

Reply:
Sounds good. If you like I can add a commit to get you started but it's difficult for me to test the changes.

Reply:
I had to go all in and just strip everything out to make it work but hey, it compiles on my end.

Reply:
Ok, I just realized there's page sized arrays/slices. Gonna fix that.